exactonline/http.py

```diff
@@ -81,7 +81,8 @@ class Options(object):
     # Do we validate the SSL certificate.
     verify_cert = False
     # What we use to validate the SSL certificate.
-    cacert_file = '/etc/ssl/certs/ca-certificates.crt'
+    # cacert_file = '/etc/ssl/certs/ca-certificates.crt'
+    cacert_file = '/etc/ssl/certs/ca-bundle.crt'
```

Comment:
Does not belong in python3.4 fixes commit. Nor is it a fix that works for everyone.

> Don’t use /etc/ssl/certs/ca-bundle.crt, even though it does probably exist on most current Fedora/RHEL installs; it’s not a canonical location for Fedora/RHEL and it’s unlikely to exist on anything else, and it may possibly go away on Fedora in future. Just don’t use it.

> So, if you wanted to handle trust store locations yourself, you could probably cover all or most Linux distros by checking for /etc/pki/tls/certs/ca-bundle.crt and /etc/ssl/certs in that order and using the first found, in the appropriate way (the first as a bundle file, the second as a directory). But you still wouldn’t be covering non-platform builds of OpenSSL, or OS X, or Windows.